test/redundantaudioencoder/RedundantAudioEncoder.test.ts

```diff
@@ -541,6 +542,57 @@ describe('RedundantAudioEncoder', () => {
     });
   });
 
+  describe('maybeEnqueueAudioFrame', () => {
+    // @ts-ignore
+    let frame: RTCEncodedAudioFrame;
+    let controller: TransformStreamDefaultController;
+
+    beforeEach(() => {
+      frame = {
+        data: new ArrayBuffer(0),
+        timestamp: 0,
+        // @ts-ignore
+        getMetadata: () => {
+          // @ts-ignore
+          return {} as RTCEncodedAudioFrameMetadata;
+        },
+      };
+
+      controller = {
+        desiredSize: 0,
+        enqueue: () => {},
+        error: () => {},
+        terminate: () => {},
+      };
+    });
+
+    it('enqueues frames that are <= the audio payload size limit', () => {
+      const enqueueSpy = sinon.spy(controller, 'enqueue');
+
+      expect(enqueueSpy.calledOnce).to.be.false;
+      frame.data = new ArrayBuffer(encoder['maxAudioPayloadSizeBytes'] - 1);
```

Comment:
No way to test this without these private function accesses?

Reply:
I can try to change this to test these cases without the need for private function accesses.

Reply:
I didn't avoid the private function accesses completely since it would have required a lot more mocking that seemed overkill for this small change, but I did update the tests to more closely resemble that actual edge cases that this change would protect against.